### PR TITLE
Starting material

### PR DESCRIPTION
**Pull request by patricia-ternes**
This PR has the starting point for the SWD3 material.

The major problem of this course is the time: just one day course.
So my idea is having this consultation material, but on the day we do one project using good practices.
Then we try to introduce a workflow that incorporates things that are on this material.

Please, let me know what you think about the material and about the plan.

Note: the material is not complete, there are still things to finish, but I think it is better to have your help now.

### CODE REVIEW

**Comment by jhodrien** [comment]
I thoroughly approve of the plan.  I think it'll make the material hang together.  We just need to make sure we don't overload ourselves with too much to do, so we don't have enough time to take a step back and talk about things with people.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# A template jupyter book documentation repository
-
-This is a template repository that can be used when generating documentation using [Jupyter-books](https://github.com/executablebooks/jupyter-book) and hosting the site via GitHub pages.
+# SWD3 Course Material
 
 ## Working with this project locally
```
Comment (murphyqm) [inline README.md:4]: Tried adding a suggestion directly to line 9 but it won't let me for some reason - can we just add a `$ cd template-jb-docs` for completeness before the conda env create command?

Comment (murphyqm) [reply README.md:4]: Also change the URL to point to this rep instead of the template one I guess!

Comment (murphyqm) [reply README.md:4]: I've just realised that the attendees don't build this material locally and they will never see this, so please ignore above comments...

**Comment by Tiany1Pan** [comment]
This might be interesting:
https://edbennett.github.io/python-testing-ci/

**Pull request merged**